src/Stepper/Step.jsx

```diff
@@ -0,0 +1,336 @@
+import React, {PropTypes} from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+import Mixins from '../mixins';
+
+import {getMuiTheme} from '../styles';
+
+const Step = React.createClass({
+  propTypes: {
+    children: PropTypes.node,
+
+    /**
+     *Override the inline-styles of div which contains all the children include control button groups
```

Comment:
Sorry! This is a Nitpick, could you add a space in each line before starting the comment? :grin: 

```
/**
 *Good
 */

/**
 * Better
 */
```